docs/src/main/sphinx/sql/show-create-function.md

```diff
@@ -2,20 +2,60 @@
 
 ## Synopsis
 
-```text
+```sql
 SHOW CREATE FUNCTION function_name
 ```
 
 ## Description
 
-Show the SQL statement that creates the specified function.
+Shows the SQL statement that created the specified function.
 
 ## Examples
 
-Show the SQL that can be run to create the `meaning_of_life` function:
+List all SQL routines and plugin functions in the `default` schema of the
```

Comment:
I dont think plugin functions are in a schema .. so just change this to `SQL catalog routines`